### PR TITLE
Removed (almost) all Try examples in documentation/comment blocks

### PR DESCRIPTION
**Pull request by kylegoetz**
As per #1916 

The only sample code/usages/tutorials in the codebase I can find now that uses Try is in `modules/docs/arrow-examples`. The code there is a port of a video that does not appear to exist at the provided URL anymore, so I don't even know an appropriate way to refactor the code (there's some references to stage 1, 2, etc.), and then some of it is just beyond my experience level right now.

I welcome any help so this can get ready.

### CODE REVIEW

**Comment by kylegoetz** [comment]
It seems clear that the sample code, which involves hitting a repo (using `Try`), it should be refactored as `IO` but when the code starts refactoring them into `Kind`, I start running into issues because I'm not there yet skill-wise.

**Comment by kylegoetz** [comment]
err, in `docs/arrow-examples/src/test/kotlin/arrow/typeclasses`. I do not know the pedagogy of the talk, if my refactoring would introduce something earlier than the speaker (whose video I can't view to confirm) intended, and at some point I hit a wall anyway. I think I refactored something as an `IO<Either<A, B>>` and then when having to refactor to `Kind` I ran into issues about the Kind-edness expected.

File: modules/docs/arrow-docs/docs/docs/arrow-optics/iso/README.md [hunk 1]
```diff
@@ -60,16 +60,16 @@ We can do the same with a Functor mapping.
...
+pointIsoTuple.modifyF(Option.functor(), point) {
+    Some((tuple.a / 2) toT (tuple.b / 2))
 }
 ```
 
 ```kotlin:ank
-val liftF: (Point2D) -> TryOf<Point2D> = pointIsoTuple.liftF(Try.functor()) {
-    Try { (tuple.a / 2) toT (tuple.b / 0) }
+val liftF: (Point2D) -> OptionOf<Point2D> = pointIsoTuple.liftF(Option.functor()) {
+    try { Some((tuple.a / 2) toT (tuple.b / 0)) } catch(e:Throwable) { None }
```
Comment (pakoito) [inline modules/docs/arrow-docs/docs/docs/arrow-optics/iso/README.md]: Option is lossy here, use Either instead.

Comment (kylegoetz) [reply modules/docs/arrow-docs/docs/docs/arrow-optics/iso/README.md]: Done.

**Comment by aballano** [comment]
This PR also fixes #2041

**Comment by rachelcarmena** [comment]
Hi! Libraries have been moved to the new repositories. Please, find the steps here: https://github.com/arrow-kt/arrow/blob/master/docs/move-to-multi-repo/how-to-move-a-pr.md and let me know if I can help you to move this PR to the correspondent repository. Thanks for your collaboration!!

**Pull request closed without merging**